test/verify/check-system-info

```diff
@@ -281,12 +282,17 @@ class TestSystemInfo(MachineCase):
         self.assertIn("Mon Jun  4 06:34:", m.execute("date"))
         self.assertIn("EEST 2018\n", m.execute("date"))
 
-    def testTimeServers(self):
+    @skipImage("timesyncd not available", "rhel-9-2", "rhel-8-8", "rhel4edge", "centos-8-stream")
+    def testTimeServersTimesyncd(self):
         m = self.machine
         b = self.browser
 
-        # Fedora/RHEL/CentOS use chrony
-        uses_timesyncd = m.image.startswith("debian") or m.image.startswith("ubuntu") or m.image == "arch"
+        if m.image.startswith("debian") or m.image.startswith("ubuntu") or m.image == "arch":
+            # timesyncd is default
+            pass
+        else:
+            # give priority to timesyncd
```

Comment:
Do you mind keeping the "# Fedora uses chrony by default" comment here for clarity?

Reply:
Done!